sphinx/themes/classic/static/classic.css_t

```diff
@@ -153,9 +154,39 @@ div.sphinxsidebar input {
 }
 
 {% if theme_collapsiblesidebar|tobool %}
+{% if theme_rightsidebar|tobool %}
```

Comment:
This also looks unrelated -- likely from separate work to add a right sidebar to the classic theme?